src/test/java/org/elasticsearch/search/highlight/HighlighterSearchTests.java

```diff
@@ -1914,8 +1879,8 @@ public void testPostingsHighlighter() throws Exception {
 
         logger.info("--> searching on _all, highlighting on field1");
         source = searchSource()
-                .query(termQuery("_all", "test"))
-                .highlight(highlight().field("field1").preTags("<xxx>").postTags("</xxx>"));
+                .query(termQuery("field1", "test"))
```

Comment:
because if we keep _all and highlight field1 there we get no output back. I didn't rely on requireFieldMatch for this one as the postings highlighter is soon not going to support it anymore, see #11077